Composer/packages/client/src/recoilModel/atoms/authState.ts

```diff
@@ -0,0 +1,55 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+import { AzureTenant, CurrentUser } from '@botframework-composer/types';
+import { atom } from 'recoil';
+
+const getFullyQualifiedKey = (value: string) => {
```

Comment:
why can't you use the existing util function that does this used in other places?

Reply:
.... this is copypasta from the other xState files, where it is defined.... I THINK to give those variables some different namespace? I had not done this, so I updated it here... but ..we might want to consider consolidating this.